apps/coscrad-frontend/src/components/resources/photographs/photograph-detail.full-view.presenter.tsx

```diff
@@ -1,14 +1,22 @@
-import { ICategorizableDetailQueryResult, IPhotographViewModel } from '@coscrad/api-interfaces';
-import './photograph-detail.full-view.presenter.css';
+import {
+    ICategorizableDetailQueryResult,
+    IPhotographViewModel,
+    ResourceType,
+} from '@coscrad/api-interfaces';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceDetailFullViewPresenter } from '../../../utils/generic-components/presenters/detail-views';
+import { FullImageView } from '../../../utils/generic-components/presenters/full-image-view';
 
 export const PhotographDetailFullViewPresenter = ({
     id,
     imageURL,
-}: ICategorizableDetailQueryResult<IPhotographViewModel>): JSX.Element => (
-    <div className="photograph-detail-container" data-testid={id}>
-        <h3>Photograph {id}</h3>
-        <div className="detail-image-container">
-            <img src={imageURL} alt={id} />
-        </div>
-    </div>
-);
+}: ICategorizableDetailQueryResult<IPhotographViewModel>): JSX.Element => {
+    const name = 'Photograph 1';
+
+    return (
+        <ResourceDetailFullViewPresenter name={name} id={id} type={ResourceType.photograph}>
+            <FullImageView imageUrl={imageURL} alt={name} />
+            <SinglePropertyPresenter display="Photograph ID" value={id} />
```

Comment:
We talked about different options for having a more user-friendly mapped sequential ID for the UI but agreed that if we go that route it would be down the road.  This is a task that is part of this story: https://www.pivotaltracker.com/story/show/184132169